website/docs/api/cli.mdx

```diff
@@ -249,6 +251,93 @@ $ python -m spacy init labels [config_path] [output_path] [--code] [--verbose] [
+time and reduced hardware requirements, and then use this command to convert
+your pipeline to use a transformer once you've verified a proof of concept. This
+can also help isolate whether any training issues are transformer-related or
+not.
+
+```cli
+$ python -m spacy configure transformer [base_model] [output_file] [--transformer_name]
+```
+
+| Name               | Description                                                                                                                                                                                                                           |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `base_model`       | A trained pipeline to resume training (package name or path). ~~str (positional)~~                                                                                                                                                    |
+| `output_file`      | Path to output `.cfg` file or `-` to write the config to stdout (so you can pipe it forward to a file or to the `train` command). Note that if you're writing to stdout, no additional logging info is printed. ~~Path (positional)~~ |
+| `transformer_name` | The name of the base HuggingFace model to use. Defaults to `roberta-base`. ~~str (option)~~                                                                                                                                           |
+
+### configure tok2vec {id="configure-tok2vec", tag="command"}
+
+Modify the base model config to use a CNN tok2vec component. Useful for
+generating a config from a transformer-based model for faster training
+iteration.
+
+```cli
+$ python -m spacy configure tok2vec [base_model] [output_file]
+```
+
+| Name          | Description                                                                                                                                                                                                                           |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `base_model`  | A trained pipeline to resume training (package name or path). ~~str (positional)~~                                                                                                                                                    |
+| `output_file` | Path to output `.cfg` file or `-` to write the config to stdout (so you can pipe it forward to a file or to the `train` command). Note that if you're writing to stdout, no additional logging info is printed. ~~Path (positional)~~ |
+
+## merge {id="merge", tag="command"}
```

Comment:
High-level question: what's the advantage of merging over chaining your pipelines?

Reply:
Good question! I assume by "chaining" you mean `nlp2(nlp1(text))`? 

The first thing a user notices is convenience - a single pipeline can be packaged as a single file. Things like `nlp.pipe` will also work correctly, while they would be weird with chained pipelines. These are pretty big advantages. 

Another, less obvious advantage but also significant advantage is shared vocabs, which should reduce memory usage.

It's out of scope for this PR, but if duplicated and frozen components can be recognized, duplicates there can be eliminated, further saving on memory usage and processing time.

Reply:
Yes, that's what I meant. Thanks for the summary!